open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/envoy/InfinityEnvoy.java

```diff
@@ -237,8 +237,18 @@ private SearchRequest createSearchRequest(QueryParameters parameters)
                     || DynamicEnumerationKey.class.isAssignableFrom(parameters.getBinFieldType()))
             {
                 field += ".keyword";
+                request.setAggs(new Aggs(field, 10000, InfinityUtilities.MISSING_VALUE));

```

Comment:
Can you put the number into a constant? It's difficult to discern why 10,000 is the chosen value for the defaults.

Reply:
(and it looks like it was that way before your changes)